### ISSUE TITLE
 `isahc` Timeout logs are spammy

### ISSUE THREAD

**Issue by nlordell** [gnosis/dex-services#1153]
It looks like `isahc` timeout logs are pretty spammy:

![image](https://user-images.githubusercontent.com/4210206/87943059-cd399280-ca9d-11ea-97ec-daf31adb083f.png)

Specifically, here is the line logging the message:
https://github.com/sagebind/isahc/blob/b99025f22740d98e9aee8a24cee09343be8ad346/src/handler.rs#L253

It is unclear to me whether or not this is a bug, as there does not appear to be other log messages indicating HTTP requests being made in that volume. My best guess is that this might be price estimates, but I am unsure.

**Comment by fleupold** [comment]
It might be the token info fetching. I also noticed that we fetch past events and token info twice in parallel when debugging the issue from last week. Will create a separate task for that.

**Comment by nlordell** [comment]
> It might be the token info fetching

When I checked I didn't see any web3 transport logs, which is why I assumed price estimation.

**Comment by nlordell** [comment]
This issue appears to have been fixed.
